lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1242,6 +1237,39 @@ def get_service_name_from_init_system(self, service):
             else:
                 return service
 
+    def get_magma_services(self) -> List[str]:
+        """
+        Returns a list of all services managed by magmad and additionally
+        magmad itself, openvswitch-switch and sctpd.
+        """
+        non_magmad_services = [
+            'magmad',
+            'sctpd',
+        ]
+
+        systemd_only_magma_services = [
+            'openvswitch-switch',
```

Comment:
This checks for openvswitch only if we use systemd, but we have this service also running when using the dockerized AGW. Is this intended to only check for it when using systemd?
Also, what about `magma_dp@envoy.service`, which we are also using in both cases?

Reply:
`openvswitch-switch` and `magma_dp@envoy` are `systemd`-services for both init systems. At the moment the edge case where a `systemd` service is checked on a dockerized AGW (or vice versa) is not implemented at all since the health check function uses the information from `MagmadUtil.init_system()`.
I added `magma_dp@envoy` to the `systemd`-only list for now, the edge case could become a little ugly.